internal/provider/resource_tfe_team.go

```diff
@@ -274,7 +275,7 @@ func resourceTFETeamDelete(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[DEBUG] Delete team: %s", d.Id())
 	err := config.Client.Teams.Delete(ctx, d.Id())
 	if err != nil {
-		if err == tfe.ErrResourceNotFound {
+		if errors.Is(err, tfe.ErrResourceNotFound) {
```

Comment:
These changes aren't related to my PR, but they showed up in the lint and I figured I might as well address them.